zebedee-cms/src/main/java/com/github/onsdigital/zebedee/api/ListKeyring.java

```diff
@@ -63,8 +70,12 @@ public ListKeyring(final CollectionKeyring collectionKeyring, final Sessions ses
     @GET
     public Set<String> listUserKeys(HttpServletRequest request, HttpServletResponse response) throws ZebedeeException {
         checkPermission(getSession(request));
+        String src = request.getParameter("src");
+
+        CollectionKeyring keyringSrc = "central".equalsIgnoreCase(src) ? legacyKeyring : centralKeyring;
```

Comment:
This looks like it may be doing the inverse of what it should. If it matches `central`, then it's using the legacyKeyring?